packages/web/src/components/edit-collection/EditCollectionForm.tsx

```diff
@@ -63,13 +76,25 @@ export const EditCollectionForm = (props: EditCollectionFormProps) => {
   const collectionTypeName = isAlbum ? 'Album' : 'Playlist'
   const validationSchema = isAlbum ? AlbumSchema : PlaylistSchema
 
+  const handleSubmit = useCallback(
+    (values: CollectionValues) => {
+      if (is_private && !values.is_private && !isReleaseConfirmationOpen) {
+        setIsReleaseConfirmationOpen(true)
+      } else {
+        setIsReleaseConfirmationOpen(false)
```

Comment:
Is this line necessary?

Reply:
i can double check if that modal closes automatically, probably does since the page naviagtes huh?